src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -202,7 +224,12 @@ export async function htmlLanguageFeaturesExtensionReady() {
 }
 
 async function razorExtensionReady() {
-    await vscode.commands.executeCommand('extension.razorActivated');
+    await vscode.commands.executeCommand('extension.razorActivated')
+        .then(async () => {
```

Comment:
What's with the love for `.then`? It seems like we've been using await everywhere else? Do I just not understand what `then` is?

Reply:
Yes, I needed a way to handle rejected promises and then was a way to do that. But I recently came to know from @NTaylorMullen that try catching await is another way to handle that. I'll do that instead.